connector/src/test/scala/com/basho/riak/spark/rdd/timeseries/TimeSeriesWriteTest.scala

```diff
@@ -231,6 +231,75 @@ class TimeSeriesWriteTest extends AbstractTimeSeriesTest(false) with AbstractRDD
       """.stripMargin, stringify(data))
   }
 
+  @Test
+  def dataFrameWriteEmptyTime(): Unit = {
```

Comment:
let's rename to bring more light on the expected behavior, kinda

dataFrameFailOnWritingEmptyTime